### PR TITLE
RELATED:RAIL-4600 update documentation for version 8.12.0 

### PR DESCRIPTION
**Pull request by ivanmjartan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1256/ : SUCCESS in 1m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/847/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1257/ : SUCCESS in 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/848/ : SUCCESS in 8s

File: website/versioned_docs/version-8.12.0/02_start__using_boilerplate.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Internet Explorer
+Internet Explorer 11 is [no longer supported](https://help.gooddata.com/pages/viewpage.action?pageId=86775029).
+
+## How to use `@gooddata/create-gooddata-react-app`
+
+To start, go to [https://github.com/gooddata/gooddata-create-gooddata-react-app](https://github.com/gooddata/gooddata-create-gooddata-react-app), and follow the instructions in the README file.
+The created application contains its own README with further information about configuration, build and deployment options.
+
+**NOTE:** This tutorial assumes that you already have an existing GoodData platform or GoodData Cloud account or a GoodData.CN installation.
```
Comment (Mara3l) [inline website/versioned_docs/version-8.12.0/02_start__using_boilerplate.md:37]: Wouldn't it be better to link each of the options?

GoodData Cloud - https://www.gooddata.com/developers/cloud-native/doc/cloud/deploy-and-install/cloud/

GoodData.CN - https://www.gooddata.com/developers/cloud-native/doc/cloud/deploy-and-install/cloud-native/

GoodData Platform - ..?

File: website/versioned_docs/version-8.12.0/10_vis__geo_pushpin_chart_component.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    separators: {
+        thousand: ",",
+        decimal: "."
+    }
+}
+```
+
+## Mapbox token provider
+
+The token provider allows setting of the Mapbox token for whole app or for a part of the app. It is not necessary to provide the token for every chart, insight or dashboard.
```
Comment (Mara3l) [inline website/versioned_docs/version-8.12.0/10_vis__geo_pushpin_chart_component.md:167]: ... token for >the< whole app ..

File: website/versioned_docs/version-8.12.0/10_vis__insight_view.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+<div style={{ height: 400, width: 600 }}>
+    <InsightView
+        insight={uriRef("<visualization-uri>")}
+        config={<chart-config>}
+    />
+</div>
+```
+**NOTE**: Make sure to also include CSS styles for the visualizations you expect to appear in the InsightView. See the [integration guide](06_cloudnative__integration.md#step-2-include-styles) for more information.
+
+**NOTE**: A set of options in `config` is different for each type of visualization. The type expected on InsightView is the same as the type of the configuration expected on the respective React component.
```
Comment (Mara3l) [inline website/versioned_docs/version-8.12.0/10_vis__insight_view.md:39]: The options in `config` are different for each visualization type. The type expected in InsightView is the same as the expected configuration type in the respective React component.

File: website/versioned_docs/version-8.12.0/10_vis__insight_view.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+## Filters
+
+For more information, see [Filter Visual Components](30_tips__filter_visual_components.md).
+
+## Caching
+
+To properly render the referenced table or chart, the InsightView component needs additional information from the GoodData platform. This information is usually static. To minimize the number of redundant requests and reduce the rendering time, some static information (such as the list of visualization classes, the color palette, or feature flags for each workspace) is cached for all InsightView components in the same application.
+
+The amount of the cached information does not impact performance in any way. However, you can manually clear the cache whenever needed (for example, after logging out, when switching workspaces or leaving a page with visualizations using the GoodData.UI components).
```
Comment (Mara3l) [inline website/versioned_docs/version-8.12.0/10_vis__insight_view.md:69]: The amount of >< cached information does not impact ...

File: website/versioned_docs/version-8.12.0/19_webcomponents_intro.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+```
+
+> The **Web Components** library is using **GoodData.UI** under the hood. 
+>
+>It is loading React and all the necessary dependencies. However, it runs in the isolated scope that will not conflict with other JavaScript running in your app.
+
+## Choosing the right embedding option
+
+GoodData provides several options for embedding, such as **iframe embedding** for dashboards or the  **GoodData.UI React library** for dashboards and insights. The Web Components library is the middle ground between those two options. It is
+more flexible than iframe embedding, yet simpler to integrate comparing to the React library.
```
Comment (Mara3l) [inline website/versioned_docs/version-8.12.0/19_webcomponents_intro.md:32]: I would omit the page break and change "It is more flexible ..." to "Web Components are more flexible..."

**Review by Mara3l — APPROVED** [review]

File: website/versioned_docs/version-8.12.0/10_vis__geo_pushpin_chart_component.md (hunk 2 above)
Comment (ivanmjartan) [reply website/versioned_docs/version-8.12.0/10_vis__geo_pushpin_chart_component.md:167]: fixed

File: website/versioned_docs/version-8.12.0/10_vis__insight_view.md (hunk 3 above)
Comment (ivanmjartan) [reply website/versioned_docs/version-8.12.0/10_vis__insight_view.md:39]: fixed

File: website/versioned_docs/version-8.12.0/19_webcomponents_intro.md (hunk 5 above)
Comment (ivanmjartan) [reply website/versioned_docs/version-8.12.0/19_webcomponents_intro.md:32]: fixed

File: website/versioned_docs/version-8.12.0/02_start__using_boilerplate.md (hunk 1 above)
Comment (ivanmjartan) [reply website/versioned_docs/version-8.12.0/02_start__using_boilerplate.md:37]: I will let it with out changes

File: website/versioned_docs/version-8.12.0/10_vis__insight_view.md (hunk 4 above)
Comment (ivanmjartan) [reply website/versioned_docs/version-8.12.0/10_vis__insight_view.md:69]: fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1258/ : SUCCESS in 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/849/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1260/ : SUCCESS in 1m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/851/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1261/ : SUCCESS in 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/852/ : SUCCESS in 4s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1262/ : SUCCESS in 1m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/178/ : SUCCESS in 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/373/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul/15/ : FAILURE in 6m 13s (non-voting)

**Pull request merged**